### DISCUSSION TITLE
How to find the Gogs version

### DISCUSSION THREAD

**Discussion by lianzxu** [gogs/gogs#7384] [Q&A] [answered]
It is kind of embarrassing to post such a question. But I really can't figure it out how to get gogs version. 

I pulled the docker image: "FROM gogs/gogs" and built a microservice with it. Somehow ssh server stops working on Gogs pod. Any ssh request to Gogs ended with "Operation timeout". However, it is still working on the pods deployed some time back. I would like to compare the version of Gogs on the "good" pods and "bad" pods.

I found this on Gogs FAQ page but I couldn't find the mentioned directory or file:

**How do I find the current version of Gogs?
The current version of Gogs is written in plain text in the file templates/.VERSION.**

Can anyone provide me some pointers? Thank you in advance.

**Comment by annpocoyo** [comment, marked as answer]
Go into your admin panel and at the very top there is the gogs version.

**Reply by lianzxu** [reply]
Got it! Thank you.
